rest_lights.cpp

```diff
@@ -1824,6 +1834,24 @@ int DeRestPluginPrivate::setWindowCoveringState(const ApiRequest &req, ApiRespon
             rspItemState[QString("/lights/%1/state/lift").arg(id)] = targetLift;
             rspItem["success"] = rspItemState;
             rsp.list.append(rspItem);
+            
+            
+            // I m using this code only for Legrand ATM but can be used for other device.
+            // Because the attribute reporting take realy long time to be done, can be 2 minutes
+            // Or it can be changed only after this time, so using an read attribute don't give usable value
+            // And can cause issue on some third app
+            if ((taskRef.lightNode->modelId() == QLatin1String("Shutter switch with neutral")) ||
+                 (taskRef.lightNode->modelId() == QLatin1String("Shutter SW with level control")) )
+            {
+                taskRef.lightNode->setValue(RStateLift, 50);
+                taskRef.lightNode->setValue(RStateBri, 127);
+                //taskRef.lightNode->setValue(RStateLift, targetLift);
+                //taskRef.lightNode->setValue(RStateBri, targetLift * 254 / 100);
+                //bool on2 = targetLift > 0;
+                //bool open2 = targetLift < 100;
+                //taskRef.lightNode->setValue(RStateOpen, open2);
+                //taskRef.lightNode->setValue(RStateOn, on2);

```

Comment:
Please remove commented out dead code, usually it doesn't age well :)

Reply:
done